### PR TITLE
AP_MSP:mod osdtest params to ease use

### PR DESCRIPTION
**Pull request by Hwurzburg**
and improve display

less steps in forthcoming wiki use page now

### CODE REVIEW

File: libraries/AP_MSP/Tools/osdtest.parm [hunk 1]
```diff
@@ -1,8 +1,11 @@
+OSD_TYPE         3.00
+SERIAL2_PROTOCOL 33.0
```
Comment (tridge) [inline libraries/AP_MSP/Tools/osdtest.parm:2]: the reason I didn't include the SERIAL2_PROTOCOL setting is this set of params can also be used on a real board which will likely be on a different serial port

Comment (Hwurzburg) [reply libraries/AP_MSP/Tools/osdtest.parm:2]: the wiki talks about that in the general config section, but this simplifies the SITL....

Comment (Hwurzburg) [reply libraries/AP_MSP/Tools/osdtest.parm:2]: added the note in  wiki PR #3000

Comment (Hwurzburg) [reply libraries/AP_MSP/Tools/osdtest.parm:2]: closing in light of the new --msposd option

**Pull request closed without merging**